packages/alert-banner/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { useEffect, useState } from 'react'
+import cookie from 'js-cookie'
+import slugify from 'slugify'
+import classNames from 'classnames'
+import VisuallyHidden from '@reach/visually-hidden'
```

Comment:
I've added `VisuallyHidden` from ReachUI to replace one-off `.visuallyHidden` styles.